contrib/cirrus/lib.sh

```diff
@@ -244,6 +245,28 @@ use_netavark() {
     # N/B: The CNI packages are still installed and available. This is
     # on purpose, since CI needs to verify the selection mechanisms are
     # functional when both are available.
+
+    # See ./contrib/cirrus/CIModes.md.
+    # Vars defined by cirrus-ci
+    # shellcheck disable=SC2154
+    if [[ "$CIRRUS_CHANGE_TITLE" =~ CI:[AN]V[AN]V= ]]; then
+        # shellcheck disable=SC2154
+        if [[ "$CIRRUS_PR_DRAFT" != "true" ]]; then
+            die "Magic 'CI:NVAV=*' string can only be used on DRAFT PRs"
+        fi
+
+        magickind=$(sed -r -e 's~(.*CI:[AN]V[AN]V=)(\w+)(.*)~\2~' <<<"$CIRRUS_CHANGE_TITLE")
+        if [[ "$magickind" == "update" ]]; then
+            warn "Updating released netavark/aardvark RPM packages from updates repo."
```

Comment:
I still find the wording confusing between these two messages (this and line 263). Sure, they look different side by side, but someone reading the log without benefit of both lines may have a hard time. Maybe something like this?
```bash
    warn "Updating ... from **fedora updates repo**"
...
    warn "Updating ... from **COPR build of main git branch**"
```

Reply:
Actually this reminded me.  We're **not** only updating from `updates`, sometimes we pull from `updates-testing`.  Let me make that clear as well...